### PR TITLE
Add member image for Josh Lipton

### LINKED ISSUE #311: Add member image for Josh Lipton

### PR DESCRIPTION
**Pull request by ctc-devops**
Authors:
 Closes #311

### CODE REVIEW

File: website/src/app/views/our-team/members.js [hunk 1]
```diff
@@ -246,6 +247,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/kei-asakawa/',
         imageURL: kei,
       },
```
Comment (alanchangxyz) [inline website/src/app/views/our-team/members.js:249]: dev/designer names should be in alphabetical order - j < k

**Review by alanchangxyz — CHANGES_REQUESTED** [review]
just an ordering issue to fix - also, can you rename your image file so that it has no underscore between your first/last name (if anything our files are named using `camelCase` but that's not necessary here)

**Review by claudeisakeeb — CHANGES_REQUESTED** [review]
Hi Josh, your changes look good. There's still one super minor merge conflict though, could you please take a look?

**Review by claudeisakeeb — APPROVED** [review]
Looks great, good job!

LGTM

**Pull request merged**